server/views/pages/adjudication.njk

```diff
@@ -49,7 +49,8 @@
               },
               {
                 key: { text: "What happened" },
-                value: { text: data.adjudication.incidentStatement.statement }
+                value: { text: data.adjudication.incidentStatement.statement },
+                classes: 'sentry-mask'
```

Comment:
You can add data attributes I believe rather than classes?

Reply:
There's custom fields I think?

Reply:
I can't seem to figure this one out to use data attributes - I've looked in the template for govukSummaryList and it doesn't look to take any other params other than classes for the rows. It's possible to add data attributes for the entire block, but not just a single row 🤔 unless I'm missing something (possibly!) 😅 

```<div class="govuk-summary-list__row {%- if anyRowHasActions and not row.actions.items %} govuk-summary-list__row--no-actions{% endif %} {%- if row.classes %} {{ row.classes }}{% endif %}">
    <dt class="govuk-summary-list__key {%- if row.key.classes %} {{ row.key.classes }}{% endif %}">
      {{ row.key.html | safe | trim | indent(6) if row.key.html else row.key.text }}
    </dt>
    <dd class="govuk-summary-list__value {%- if row.value.classes %} {{ row.value.classes }}{% endif %}">
      {{ row.value.html | safe | trim | indent(6) if row.value.html else row.value.text }}
    </dd>
    {% if row.actions.items.length %}
    <dd class="govuk-summary-list__actions {%- if row.actions.classes %} {{ row.actions.classes }}{% endif %}">
      {% if row.actions.items.length == 1 %}
        {{- _actionLink(row.actions.items[0], params.card.title) | trim | indent(6, true) }}
      {% else %}
      <ul class="govuk-summary-list__actions-list">
        {% for action in row.actions.items %}
        <li class="govuk-summary-list__actions-list-item">
          {{ _actionLink(action, params.card.title) | trim | indent(8) }}
        </li>
        {% endfor %}
      </ul>
      {% endif %}
    </dd>
    {% endif %}
  </div>```

Reply:
Looks like you can use it here:

```
      actions: {
        items: [
          {
            href: "#",
            text: "Change",
            visuallyHiddenText: "name"
          }
        ]
      }
```

from the docs? https://design-system.service.gov.uk/components/summary-list/ Not sure if that's what you want?

Reply:
Managed to do it using the html field instead of text :)